planting/planting_api_v1.py

```diff
@@ -76,10 +61,20 @@ def __init__(self):
                            syntax=None)
         self.loader = DataLoader()
         self.variable_manager = VariableManager()
-        self.passwords = dict()
+        self.passwords = {"conn_pass": env.password}
         self.results_callback = ResultCallback()
+        self.logger = logger
+        level = logger.DEBUG
+        complete_log = []
+        logger.add_consumers(
+            (logger.VERBOSE_DEBUG, sys.stdout),
+            (level, complete_log.append)
+        )
         # after ansible 2.3 need parameter 'sources'
-        self.inventory = InventoryManager(loader=self.loader, sources='hosts')
+        # create inventory, use path to host config file
+        # as source or hosts in a comma separated string
+        self.inventory = InventoryManager(
+            loader=self.loader, sources=env.ip+',')
         self.variable_manager = VariableManager(
             loader=self.loader, inventory=self.inventory)
```

Comment:
l have modified the host and password register, so you should add the code
`host_info = Host(name=env.ip, port='22')
        self.variable_manager.set_host_variable(
            host_info, 'ansible_user', env.remote_user)
        self.variable_manager.set_host_variable(
            host_info, 'ansible_pass', env.password)`